Solon_Burleson/RunCode.py

```diff
@@ -1,7 +1,58 @@
-import random
-def randInt(max=0, min=0):
-    if max == 0 and min == 0:
-        print(int(random.random()*100))
-    else:
-        print(int(random.random()*(max-min)+min))
-randInt(max=100,min=50)
+class Node:
+    def __init__(self, value):
+        self.value = value
+        self.next = None
+
+class Slist:
+    def __init__(self,value):
+        node = Node(value)
+        self.head = node
+
+    def addNode(self, value):
+        node = Node(value)
+        runner = self.head
+        while runner.next != None:
+            runner = runner.next
+        runner.next = node
+
+    def removeNode(self, value):
+        runner = self.head
+        if runner.value == value:
+            runner = runner.next
+            self.head = runner
+        prevrunner = runner
+        while runner.next != None:
+            if runner.value == value:
+                prevrunner.next = runner.next
+            prevrunner = runner
+            runner=runner.next
+        if runner.value == value and runner.next == None:
+            prevrunner.next = None
+
+    def insertNode(self, value, index):
+        count = 0
+        runner = self.head
+        if index == 0:
+            self.head = Node(value)
+            self.head.next = runner
+        prevrunner = runner
+        while runner.next!= None:
+            if count == index:
+                prevrunner.next = Node(value)
+                prevrunner.next.next = runner
```

Comment:
return after the if condition //we stop the function as soon as the insertion is done (task completed)